assets/js/vue/ResearchGroupApp.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div class="container-xl" ref="formContainer">
+    <div class="container" ref="formContainer">
```

Comment:
This makes the background narrower than before. The container is also really tight around the content. For example the cards, and the results per page selector. Maybe it should be wider, or need padding?

Reply:
Sorry, i dint notice the background color thing. I fixed it, not sure about the content being tight. I haven't changed any of that, and one of the things in the ticket was to reduce padding

Reply:
I'm not seeing where this is really tight around content. It looks fine in both Firefox and Chrome to me on the search page.

Reply:
> I'm not seeing where this is really tight around content. It looks fine in both Firefox and Chrome to me on the search page.

That's been fixed now.